src/main/java/seedu/address/model/module/ModuleHasModuleCodePredicate.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.model.module;
+
+import java.util.function.Predicate;
+
+/**
+ * Tests that a {@code Module}'s {@code ModuleCode} is the same as the given {@code ModuleCode}
+ */
+public class ModuleHasModuleCodePredicate implements Predicate<Module> {
```

Comment:
hmm maybe we can name this HasModuleCodePredicate instead? slightly shorter and we can keep to HasXXXPredicate instead of ModuleHasXXXPredicate. what do u think?